sdv/single_table/base.py

```diff
@@ -99,6 +99,18 @@ def _update_default_transformers(self):
             for sdtype, transformer in self._model_sdtype_transformers.items():
                 self._data_processor._update_transformers_by_sdtypes(sdtype, transformer)
 
+    def _check_original_metadata_updated(self):
+        if isinstance(self._original_metadata, Metadata):
+            metadata = self._original_metadata._convert_to_single_table()
+        else:
+            metadata = self._original_metadata
```

Comment:
since this is called in sampling, we need to be backwards compatible. We should safeely access self._original_metadata as it won't exist on an old saved synthesizer

Reply:
Added here https://github.com/sdv-dev/SDV/pull/2472/commits/f81be7b2dcb55231aa2f2a4d6d17f0cb547d4005